README.md

```diff
@@ -18,6 +21,7 @@ dev-server watch
 
 **Note for windows users:**  
 If dev-server command is not found, check that the npm directory (typically ;C:\Users\%username%\AppData\Roaming\npm) is included into the PATH variable
+Users should run ```npm run dev-server``` rather than just ```dev-server```, as the latter may not work directly on Linux systems without additional setup (such as adding it to the PATH).
```

Comment:
Combined with the remainder of the readme, this is confusing. `npm run dev-server` only works if it was added as a devDependency. If we want to keep this sentence, it should be moved to the dev-dependency section.